docs/DeveloperGuide.md

```diff
@@ -467,14 +467,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
         Use case resumes at step 2.
     
-**Use case: Calculate an employee's salary**
+**Use case: Calculate an employee's hourlySalary**
 
 **MSS**
 
 1.  User requests to list employees
 2.  HeRon shows a list of employees
-3.  User requests to calculate the salary of a specific employee in the list
-4.  HeRon calculates and displays the salary of that employee
+3.  User requests to calculate the hourlySalary of a specific employee in the list
+4.  HeRon calculates and displays the hourlySalary of that employee
```

Comment:
I think this should be just plain English (e.g. "hourly salary"). If not it should be just "salary". A global find and replace might have been accidentally done.